src/json.rs

```diff
@@ -30,9 +32,8 @@ pub enum JsonError {
 
 #[derive(Debug, Ord, PartialOrd, Eq, PartialEq)]
 pub enum JsonOutOfRangeError {
```

Comment:
Think that this should be `JsonError`, or keep it the same and change the enums to just be `U8` & `U128`.

Reply:
AFAIR we intentionally added a separate sub-enum for the `JsonOutOfRangeError`. What's the point of removing it?

Reply:
Yes, it was special extended casef or error kind.